app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -110,12 +110,13 @@ $ ->
 
       drawControls: ->
         super()
-        @drawGroupControls()
-        @drawYAxisControls()
+        @drawGroupControls(data.textFields)
+        @drawYAxisControls('Y Axis', globals.configs.fieldSelection,
```

Comment:
I ended up changing this, then realized that it's called with something other than 'Y Axis' for like literally every other vis. I then added it back with a default argument haha